Packages/com.unity.inputsystem/InputSystem/Plugins/XR/GearVR.cs

```diff
@@ -4,32 +4,27 @@
 namespace UnityEngine.Experimental.Input.Plugins.XR
 {
     [InputControlLayout]
-    public class GearVRHMD : XRHMD
+    public class GearVRSimpleHMD : XRHMD
```

Comment:
Base rather than Simple?

Reply:
"Base" is probably better politically for our relationship with Oculus/Samsung